controllers/operands/kubevirt.go

```diff
@@ -47,7 +47,8 @@ const (
 )
 
 const (
-	primaryUDNNetworkBindingName = "passt"
+	primaryUDNNetworkBindingName = "l2bridge"
+	passtUDNNetworkBindingName   = "passt"
```

Comment:
Please consider dropping the `UDN` prefix, as these are just two binding plugins, and using them does not guarantee a connection to UDN.
```suggestion
	l2bridgeBindingName = "l2bridge"
	passtBindingName   = "passt"
```

Same bellow.

Reply:
True but we are triggering them by `primaryUserDefinedNetworkBinding` FG so it is for UDN
Moreover i prefer to do minimal changes on this PR, it is not its scope, especially cosmetics, 
once we do another cycle we can do whatever desired